CMakeLists.txt

```diff
@@ -92,8 +92,8 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS
     CACHE STRING "Enable/Disable output of compile commands during generation." FORCE
 )
 
-# Enable all warnings that we can
-add_compile_options(-Wall -Wpedantic -Wextra)
+# Enable warnings
+add_compile_options(-Wall -Wextra)
```

Comment:
Ummmmmmmm

Reply:
Apparently, the enum templates that I use are not allowed by pedantic and I don't have a specific rule I can turn off :(
Should work out what pedantic does and if any of them are actually important

Reply:
Remove now pls 😇